src/main/resources/assets/bno/lang/ru_ru.json

```diff
@@ -0,0 +1,63 @@
+    "item.bno.tin_nugget": "Оловянный самородок",
+    "item.bno.osmium_nugget": "Осмиевый самородок",
+
+    "item.bno.aluminum_ingot": "Алюминиевый слиток",
+    "item.bno.copper_ingot": "Медный слиток",
+    "item.bno.lead_ingot": "Свинцовый слиток",
+    "item.bno.nickel_ingot": "Никелевый слиток",
+    "item.bno.silver_ingot": "Серебряный слиток",
+    "item.bno.tin_ingot": "Оловянный слиток",
+    "item.bno.osmium_ingot": "Осмиевый слиток",
+    "item.bno.uranium_ingot": "Урановый слиток",
+
+    "block.bno.aluminum_block": "Алюминиевый блок",
+    "block.bno.copper_block": "Медный блок",
+    "block.bno.lead_block": "Свинцовый блок",
+    "block.bno.nickel_block": "Никелевый блок",
+    "block.bno.silver_block": "Серебряный блок",
+    "block.bno.tin_block": "Оловянный блок",
+    "block.bno.osmium_block": "Осмиевый блок",
+    "block.bno.uranium_block": "Урановый блок",
+
+    "block.bno.netheraluminum_ore": "Незеровская алюминиевая руда",
+    "block.bno.nethercopper_ore": "Незеровская медная руда",
+    "block.bno.netherlead_ore": "Незеровская свинцовая руда",
+    "block.bno.nethernickel_ore": "Незеровская никелевая руда",
+    "block.bno.nethersilver_ore": "Незеровская серебряная руда",
+    "block.bno.nethertin_ore": "Незеровская оловянная руда",
+    "block.bno.netherosmium_ore": "Незеровская осмиевая руда",
+    "block.bno.netheruranium_ore": "Незеровская урановая руда",
+
+    "block.bno.netheremerald_ore": "Незеровская изумрудная руда",
+    "block.bno.netherdiamond_ore": "Незеровская алмазная руда",
+    "block.bno.netherredstone_ore": "Незеровская руда красного камня",
+    "block.bno.netherlapis_ore": "Незеровская изумрудная руда",
+    "block.bno.nethercoal_ore": "Руда незеровского угля",
+
+    "block.bno.netheriron_ore": "Незеровская железная руда",
+
+    "tooltip.netheremerald_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netherdiamond_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netherredstone_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netherlapis_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.nethercoal_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.nethergold_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.nethersilver_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netheriron_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netherlead_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.nethernickel_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.nethercopper_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netheraluminum_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.nethertin_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netherosmium_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.netheruranium_ore.tip": "§aОбнаружен среди y{0} и y{1}",
+    "tooltip.config.tip": "§4§oГенерация мира отключена, переведено сообществом МодоВестник  - Включить через файл конфигурации"
```

Comment:
"World generation is disabled, *translated by modovestnik community* - Enable through config file"
Advertisement of his VK community, just like in any other of his "translations"

Reply:
Ah.  Thank you for catching this.  I will get it updated